extensions/binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/handler/HueBridgeHandler.java

```diff
@@ -299,6 +279,24 @@ public void onConnectionResumed(HueBridge bridge) {
             }
         }
     }
+    
+    /**
+     * Check USER_NAME config for null. Call onConnectionResumed() otherwise.
+     * @return True if USER_NAME was not null.
```

Comment:
AFAIK there should be an empty line between the description and the tag.

<code>Insert a blank comment line between the description and the list of tags, as shown.</code> (http://www.oracle.com/technetwork/articles/java/index-137868.html)